python/ray/tune/tests/test_trial_scheduler.py

```diff
@@ -143,7 +143,36 @@ def result2(t, rew):
             grace_period=0,
             min_samples_required=1,
             time_attr="training_iteration",
-            reward_attr="neg_mean_loss")
+            metric="neg_mean_loss",
```

Comment:
nit: it would be nice to reduce the amount of repeated code by making the below logic a helper method like `def _test_metrics(self, func, metric, mode): ...` and then have both `def testAlternativeMetrics` and `def testAlternativeMetricsMin` call it.

Reply:
(same with the other schedulers tested in this class)